### ISSUE TITLE
Contacts trouble

### ISSUE THREAD

**Issue by cremesk** [modoboa/modoboa-contacts#26]
Hello,

i think i have some trouble with the contacts module.

I have over 300 conntacts on my Phone and my Thunderbird Contacts Plugin.
This work good and the devices are synced.

Problem is the Webinterface. I see only 4 contacts but no longer exist and the other 300 should be there I do not see.

what can I do?

Debug mode is enable and shows no problems and errors.
In the config i have enable the modoboa_app
```
...
  'modoboa_contacts',
...
```
and on the end of file:
```
try:
    from modoboa_contacts import settings as modoboa_contacts_settings
    modoboa_contacts_settings.apply(globals())
except AttributeError:
    from modoboa_contacts.settings import *  # noqa
```

thanks for any help

**Comment by tonioo** [comment]
@cremesk Is this issue still active?

**Comment by cremesk** [comment]
@tonioo yes the issue is active.

**Comment by cremesk** [comment]
complement:

I can see all `.vcf`'s in `/srv/radicale/collections/collection-root/<account>/contacts`

**Comment by tonioo** [comment]
@cremesk How did you create this collection? Were you using it before installing modoboa?

**Comment by cremesk** [comment]
No, I didn't do anything special. Modoboa installed and configured by default.

After that I made the calendar and the contacts available on the cell phone and imported my contacts.

Now the status is that I have all contacts in the Radicale folder on the server.
But however, I cannot see them via the web interface.

**Comment by tonioo** [comment]
@cremesk There is a sync mechanism between Modoboa and the carddav collection but it might not be enabled in your case... Can you take a screen of the web interface please?

**Comment by cremesk** [comment]
Yes sure :)

my sync is enabled:
![modoaboa_contacts_prefs_2020-03-26_15-18](https://user-images.githubusercontent.com/7526990/77657246-4fdd9580-6f75-11ea-854d-01820d1e8766.png)

here a screenshot of my contacts:
![modoboa_contacts_2020-03-26_15-19](https://user-images.githubusercontent.com/7526990/77657300-671c8300-6f75-11ea-884e-a8f80d5d75c2.png)

**Comment by s256** [comment]
Might have the same issue.
Contacts synced from a client are not displayed in the web interface. But contacts created / modified in the web interface are synced to the client.
